src/collar/oc_titler.lsl

```diff
@@ -99,14 +97,8 @@ ShowHideTitle() {
           }
         }
         else {
-            rotation start = ZERO_ROTATION;
```

Comment:
Also needed for the new "image" command.  Have you tested your one-liner solution below with the "image" command, in a variety of different collars with different root rotations, attached to different attach points?  In developing the feature I tried several solutions like your one liner below and none were satisfactory.

Reply:
not tested this yet but there is no reason this should not work

Reply:
The big problem I found with this kind of solution is that "up", as far as the spine attachment is concerned, actually projects in front of the av instead of up over his/her head.